metaflow/plugins/argo/argo_inner_flow.py

```diff
@@ -1,31 +1,64 @@
 from metaflow.plugins.argo.argo_client import ArgoClient
 from metaflow.metaflow_config import KUBERNETES_NAMESPACE
+import time
+
+### ADDING WAIT FEATURE AND UPDATING PROPERTIES
 
 
 class TriggeredRun:
+    """
+    This class takes in information to identify and trigger a run of a
+    Metaflow flow using the Argo plugin. The run is triggered immediately 
+    upon initialization of this object, and the run is identified based on 
+    the parameter for the Metaflow flow name. Users may use additional 
+    parameters to alter the run, as well as whether to wait for the run to 
+    finish. 
+    
+    The object allows users to access information relating to the triggered 
+    run, including status ('Running', 'Failed', or 'Successful'), failed
```

Comment:
I think aligning is generally a good idea and makes sense. One issue in this case is that Metaflow's flow class has .finished and .successful, and they mean different things than what they mean for this class. I'm worried this will cause confusion. One idea is to use ".completed" instead of ".finished", so that the meanings are not misaligned between Run class and TriggeredRun class.

Reply:
Yun: use finished, and also report bug to metaflow so they can fix it in their Run class.

Reply:
done

Reply:
Taleb: Use .completed because Metaflow may not want to change .finished